compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -917,8 +919,14 @@ class Allocator {
       Primary.Options.clear(OptionBit::AddLargeAllocationSlack);
   }
 
-  const char *getStackDepotAddress() const {
-    return reinterpret_cast<const char *>(&Depot);
+  const char *getStackDepotAddress() {
+    initThreadMaybe();
```

Comment:
This is used to init thread local cache for allocation. Why do we need to call this function here?